helm/deletion-blocker-operator/templates/psp.yaml

```diff
@@ -24,6 +24,13 @@ spec:
     rule: RunAsAny
   supplementalGroups:
     rule: RunAsAny
+  volumes:
```

Comment:
do we really need/use all these volume types?

Reply:
https://gigantic.slack.com/archives/C02FL4EAADD/p1675854402433689?thread_ts=1675850424.939239&cid=C02FL4EAADD

We assume secret and projected should suffice. 
Since we do not yet clearly understand the origin of this error we've settled on allowing the same volumes as the restricted policies.

More thoughts/ideas are welcome if you have any.

Reply:
works for me